.travis.yml

```diff
@@ -23,10 +23,23 @@ jobs:
         - graphviz-dev
         - xmlstarlet
         - jq
-    env: SMING_ARCH=Host
+    env:
+    - SMING_ARCH=Host
+
   - stage: build
     os: linux
-    env: SMING_ARCH=Esp8266 SDK_VERSION=3.0.1
+    env:
+    - SMING_ARCH=Esp8266
+    - SDK_VERSION=3.0.1
+    - ESP_HOME=$TRAVIS_BUILD_DIR/opt/esp-alt-sdk
+
+  - stage: build c++17
```

Comment:
@mikee47 Can you try to use `build` as a stage name also here? It would be better to have the different builds running in parallel and not wait for `build` to finish in order for `build c++17` to start. I am not sure if travis will merge them under on stage and run them in parallel but it is worth trying.